Amplify/DevMenu/View/EnvironmentInfoDetailView.swift

```diff
@@ -0,0 +1,150 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// Detail view showing environment information
+@available(iOS 13.0.0, *)
+struct EnvironmentInfoDetailView: View {
+    private let screenTitle  = "Environment Information"
+    private let amplifyEnvSectionTitle  = "Amplify Environment Information"
+    private let devEnvSectionTitle  = "Developer Environment Information"
+
+    /// Lists containing items belonging to two sections : Amplify and Developer Environment Informatoin
+    private var amplifyEnvSectionItems = [EnvironmentInfoItem]()
+    private var devEnvSectionItems = [EnvironmentInfoItem]()
+
+    init() {
+        // TODO : Read environment information from json file and
+        // call populateDevEnvSectionItems()
+        populateAmplifyEnvSectionItems()
+    }
+
+    var body: some View {
+        SwiftUI.List {
+            Section(header: Text(amplifyEnvSectionTitle), footer: EmptyView()) {
+                if amplifyEnvSectionItems.isEmpty {
+                    NoItemView()
+                } else {
+                    ForEach(amplifyEnvSectionItems) { listItem in
+                        InfoRow(item: listItem)
+                    }
+                }
+            }
+            Section(header: Text(devEnvSectionTitle), footer: EmptyView()) {
+                if devEnvSectionItems.isEmpty {
+                    NoItemView()
+                } else {
+                    ForEach(devEnvSectionItems) { listItem in
+                        InfoRow(item: listItem)
+                    }
+                }
+            }
+        }
+        .navigationBarTitle(screenTitle)
+        .listStyle(GroupedListStyle())
+    }
+
+    private mutating func populateDevEnvSectionItems(devEnvInfo: DevEnvironmentInfo) {
```

Comment:
Lets move these data related function out from UI. Create a class similar to my example above. `EnvironmentInfoHelper?`